src/components/DatePicker/DatePicker.vue

```diff
@@ -26,6 +26,7 @@
                       v-on="on"
                       v-on:keydown="$event.preventDefault()"
                       v-on:keyup.enter="emitDate(dateText)"
+                      v-on:click:append="on.click"
```

Comment:
We might want to add this to our other date pickers... I suggest a new ticket.

Reply:
For sure! What other Date Pickers do we have? In the create UI maybe?

Reply:
Off the top of my head:
- FED in Create UI
- AR Date in Filings UI

Reply:
Okie doke will do!

Reply:
Ticket here :) : https://app.zenhub.com/workspaces/entity-5bf2f2164b5806bc2bf60531/issues/bcgov/entity/7017

Reply:
Perfect, thanks!